### DISCUSSION TITLE
Import recipe from Youtube video description

### DISCUSSION THREAD

**Discussion by akhilrex** [mealie-recipes/mealie#534] [Ideas]
A lot of food youtubers mention the recipe in the video description. It would be nice if it can be imported when a Youtube url is entered.

**Comment by zierbeek** [comment]
Hi! Since the recipes are now parsed by https://github.com/hhursev/recipe-scrapers/ , any improvements to the parser should be asked there.

**Comment by sephrat** [comment]
Is the recipe just written in plain text in the video description? Or is it a link to something?
Can you provide an example?

**Comment by akhilrex** [comment]
https://youtu.be/xqQsshnvDVo

Thanks and Regards

Akhil Gupta

**Reply by sephrat** [reply]
There's not much that can be done with plain text/unstructured recipes. Any Youtuber will have its own way of arranging ingredients, instructions and so on. I'm afraid you'll have to stick to a copy/paste process. You can still make use of "Bulk add" buttons to help with this task.
